earth_enterprise/src/fusion/autoingest/StorageManager_unittest.cpp

```diff
@@ -53,7 +53,7 @@ class TestHandle : public AssetHandleInterface<TestItem> {
     virtual const AssetKey Key() const { return name; }
     virtual string Filename() const { return "/dev/null"; }
     virtual HandleType Load(const string &) const {
-      return khRefGuardFromNew<TestItem>(new TestItem());
+      return HandleType(std::make_shared<TestItem>());
```

Comment:
Very minor: I don't think the `HandleType` wrapper around this is strictly necessary since std::make_shared returns a HandleType.